conanfile.py

```
@@ -59,4 +69,20 @@ def package_info(self):
         self.cpp_info.libdirs = ["lib"]
         self.cpp_info.includedirs = ["include", "external/imgui"]
         self.cpp_info.defines = ["IMGUI_DEFINE_MATH_OPERATORS"]
+        if self.settings.os == "Linux":
+            # System SDL2/SDL_image/SDL_ttf/GLFW — propagate link flags and
+            # SDL2 include paths to all consumers. Paths are detected via
+            # pkg-config rather than hard-coding a sysroot-relative location.
+            self.cpp_info.system_libs = ["SDL2", "SDL2_image", "SDL2_ttf", "glfw", "freetype"]
+            import subprocess
+            try:
+                flags = subprocess.check_output(
+                    ["pkg-config", "--cflags-only-I", "sdl2"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)